src/cpp/r/R/Tools.R

```diff
@@ -170,28 +170,45 @@ environment(.rs.Env[[".rs.addFunction"]]) <- .rs.Env
 })
 
 # save current state of options() to file
-.rs.addFunction( "saveOptions", function(filename)
+.rs.addFunction("saveOptions", function(filename)
 {
-   opt = options();
-   suppressWarnings(save(opt, file=filename))
+   # get reference to current options
+   opt <- options()
+   
+   # don't attempt to serialize cpp11 preserve env
```

Comment:
Maybe we should add a comment here about why and link to this issue? (so future maintainers can decide whether this workaround is safe to remove)